requirements.txt

```diff
@@ -13,14 +14,21 @@ django-oauth-toolkit==0.12.0
 django-storages==1.5.1
 djangorestframework==3.5.3
 docopt==0.6.2
+enum-compat==0.0.2
+flake8==3.3.0
+flake8-import-order==0.12
+flake8-strict==0.1.4
+flake8-tidy-imports==1.0.5
 freezegun==0.3.8
 isort==4.2.5
 lazy-object-proxy==1.2.2
-mccabe==0.5.2
+Markdown==2.6.8
```

Comment:
@just-paja And that approach doesn't preserve information, which requirements are needed and which one are their dependencies. Also with two requirement files it might be easier to update all dependencies.

But until the requirements file is too large to maintain, I am OK to stay with current system.